### ISSUE TITLE
[ResponsivePopover]: ResponsivePopover cannot be activated using enter under JAWS VPC mode

### ISSUE THREAD

**Issue by quinn-zhang-sap** [SAP/ui5-webcomponents#11943]
### Bug Description

According to this ticket: https://jira.tools.sap/browse/SFSL-15314

## Env Details
Link: https://qacand.hcm.ondemand.com/
Company: TALCPMDR
Username: adminb1
Password: pwd

## Description
When under JAWS VPC mode, the ResponsivePopover does not pop up properly.

### Expected Behaviour

The ResponsivePopover can be activated by enter.

### Steps to Reproduce

1. Login using the given instance
2. Go to 'Continuous Performance' from module switcher
3. Click 'Share Activities' button on the top right of the kanban board
4. The 'Share Activities with Another Channel' dialog will appear
5. Click the shape in the 'Select a channel' input box
6. The 'Select Channel' dialog will appear
7. Open the JAWS VPC mode and use tab navigate to the info button on the top right of the dialog
8. Press Enter and the ResponsivePopover does not pop up

### Priority

None

### UI5 Web Components Version

2.10.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by GerganaKremenska** [comment]
Hello @SAP/ui5-webcomponents-topic-p, 
 
Affected component: ui5-icon

Steps to reproduce: 
1. open [sample](https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG5cblxuXG48aGVhZD5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgXG4gICAgPHN0eWxlPlxuICAgICAgKjpub3QoOmRlZmluZWQpIHtcbiAgICAgICAgZGlzcGxheTogbm9uZTtcbiAgICAgIH1cblxuICAgIGh0bWwge1xuICAgICAgZm9yY2VkLWNvbG9yLWFkanVzdDogbm9uZTtcbiAgICB9XG4gICAgPC9zdHlsZT5cblxuICAgIFxuICAgIDxzdHlsZT5cbiAgICAgICo6bm90KDpkZWZpbmVkKSB7XG4gICAgICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgICB9XG5cbiAgICBodG1sIHtcbiAgICAgIGZvcmNlZC1jb2xvci1hZGp1c3Q6IG5vbmU7XG4gICAgfVxuICAgIDwvc3R5bGU-XG5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpXCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS1pY29uIGlkPVwiaWNvblwiIG5hbWU9XCJob21lXCIgbW9kZT1cIkludGVyYWN0aXZlXCI-PC91aTUtaWNvbj5cbiAgICA8dWk1LXJlc3BvbnNpdmUtcG9wb3ZlciBvcGVuZXI9XCJpY29uXCIgaGVhZGVyLXRleHQ9XCJOZXdzbGV0dGVyIHN1YnNjcmlwdGlvblwiPiAgICAgICAgXG4gICAgPC91aTUtcmVzcG9uc2l2ZS1wb3BvdmVyPlxuICAgIDwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuICAgIDxzY3JpcHQgdHlwZT1cIm1vZHVsZVwiIHNyYz1cIm1haW4uanNcIj48L3NjcmlwdD5cbjwvYm9keT5cblxuPC9odG1sPlxuPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuIn0sIm1haW4uanMiOnsibmFtZSI6Im1haW4uanMiLCJjb250ZW50IjoiLyogcGxheWdyb3VuZC1oaWRlICovXG5pbXBvcnQgXCIuL3BsYXlncm91bmQtc3VwcG9ydC5qc1wiO1xuLyogcGxheWdyb3VuZC1oaWRlLWVuZCAqL1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvSWNvbi5qc1wiO1xuXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9SZXNwb25zaXZlUG9wb3Zlci5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvQnV0dG9uLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9MYWJlbC5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvSW5wdXQuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2hvbWUuanNcIjtcblxuY29uc3QgaWNvbiA9IFsuLi5kb2N1bWVudC5nZXRFbGVtZW50c0J5VGFnTmFtZShcInVpNS1pY29uXCIpXVswXTtcbmNvbnN0IHJlc3BQb3BvdmVyID0gWy4uLmRvY3VtZW50LmdldEVsZW1lbnRzQnlUYWdOYW1lKFwidWk1LXJlc3BvbnNpdmUtcG9wb3ZlclwiKV1bMF07XG5cbmljb24uYWRkRXZlbnRMaXN0ZW5lcihcImNsaWNrXCIsICgpID0-IHtcblx0cmVzcFBvcG92ZXIub3BlbiA9ICFyZXNwUG9wb3Zlci5vcGVuO1xuICBjb25zb2xlLmxvZyhcImN1c3RvbSBjbGljayBmaXJlZFwiKTtcbn0pO1xuXG5cbiJ9LCJtYWluLmNzcyI6eyJuYW1lIjoibWFpbi5jc3MiLCJjb250ZW50IjoiLml0ZW0tY29udGVudCB7XG5cdGRpc3BsYXk6IGZsZXg7XG5cdGFsaWduLWl0ZW1zOiBjZW50ZXI7XG5cdGp1c3RpZnktY29udGVudDogc3BhY2UtYmV0d2Vlbjtcblx0d2lkdGg6IDEwMCU7XG5cdGhlaWdodDogMTAwJTtcbn1cblxuLmdyZWVuIHtcblx0Y29sb3I6IGdyZWVuO1xufVxuXG4uaXRlbS10aXRsZXMge1xuXHRkaXNwbGF5OiBmbGV4O1xuXHRmbGV4LWRpcmVjdGlvbjogY29sdW1uO1xufSJ9fQ) on Chrome
2. Start JAWS 2025
3. Try activating the icon with 'Space' or 'Enter'

Expected behavior: Popover is open (custom click is fired)
Actual behavior: Popover is not open

Note:  Popover is opened (custom click is fired) when Voice over is in used
Note2: Popover is not opened when NVDA on Windows is in used

**Comment by PetyaMarkovaBogdanova** [comment]
Dear @quinn-zhang-sap , 

This is a known issue - _**The sap.m.Button gets activated on SPACE key down when NVDA or JWAS in VPC mode is started.
The “keydown” event is not fired at all when NVDA or JAWS with VPC is started.**_

Since we do not have influence over it and since it is described in the wiki with Jaws issues I am closing the incident.



Best Regards,
Petya Markova

UI5 Web Components Developer
